topi/tests/python/test_topi_math.py

```diff
@@ -68,8 +68,7 @@ def check_device(device):
             foo(a, b)
             tvm.testing.assert_allclose(b.asnumpy(), b_np, rtol=1e-5, atol=1e-5)
 
-        for device in get_all_backend():
-            check_device(device)
+        check_device('llvm -device=arm-cpu')
```

Comment:
@tqchen @cowanmeg  @ZihengJiang  

If you get time can you please check on this issue as detailed below? thanks. 

-------------------------------
CI was failing for this PR,  on `test_topi_math.py`. 

However, as I understand and as I've put in the comment earlier, it shouldn't fail as this PR is not changing anything functionally. 

I've attached the stack trace below.  

During the debugging I found similar error happening on the `PyTorch` because of deconstructors called inappropriately https://github.com/pytorch/pytorch/issues/5400 . 

It seems to me this error also has something to do with deconstructor as I noticed errors happening after a call to `TVMFreeFunc` or `TVMArrayFree`. 

I experimented a bit and found that when I run each backed individually on `test_topi_math.py` , 
it all works without any errors, however, when I run all of them together it fails with following types of errors. 



```
*** Error in `python3': corrupted double-linked list: 0x000000000220a1e0 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x777e5)[0x7fb3d99de7e5]
/lib/x86_64-linux-gnu/libc.so.6(+0x80baf)[0x7fb3d99e7baf]
/lib/x86_64-linux-gnu/libc.so.6(cfree+0x4c)[0x7fb3d99eb53c]
/usr/lib/llvm-6.0/lib/libLLVM-6.0.so.1(_ZN4llvm10BasicBlockD2Ev+0x74)[0x7fb395b56bb4]
/usr/lib/llvm-6.0/lib/libLLVM-6.0.so.1(_ZN4llvm10BasicBlock15eraseFromParentEv+0x64)[0x7fb395b576b4]
/usr/lib/llvm-6.0/lib/libLLVM-6.0.so.1(_ZN4llvm8Function17dropAllReferencesEv+0x85)[0x7fb395bc67c5]
/usr/lib/llvm-6.0/lib/libLLVM-6.0.so.1(_ZN4llvm6Module17dropAllReferencesEv+0x30)[0x7fb395c114a0]
/usr/lib/llvm-6.0/lib/libLLVM-6.0.so.1(_ZN4llvm6ModuleD1Ev+0x28)[0x7fb395c11668]
/usr/lib/llvm-6.0/lib/libLLVM-6.0.so.1(_ZN4llvm5MCJITD1Ev+0x4b0)[0x7fb396c5bdc0]
/usr/lib/llvm-6.0/lib/libLLVM-6.0.so.1(_ZN4llvm5MCJITD0Ev+0x9)[0x7fb396c5c159]
/home/u00472384/repos/tvm-umang/build/libtvm.so(_ZNSt19_Sp_counted_deleterIPN3tvm7codegen14LLVMModuleNodeENSt12__shared_ptrIS2_LN9__gnu_cxx12_Lock_policyE2EE8_DeleterISaIS2_EEES9_LS6_2EE10_M_disposeEv+0x71)[0x7fb399885021]
/home/u00472384/repos/tvm-umang/build/libtvm.so(+0xba7343)[0x7fb3998af343]
/home/u00472384/repos/tvm-umang/build/libtvm.so(TVMFuncFree+0x1c)[0x7fb3998956cc]
/home/u00472384/repos/tvm-umang/python/tvm/_ffi/_cy3/core.cpython-35m-x86_64-linux-gnu.so(+0xe6a9)[0x7fb3942926a9]
python3[0x58170a]
python3[0x581a52]
python3[0x4e96b7]
python3[0x54020f]
python3(PyEval_EvalFrameEx+0x5513)[0x53bc93]
python3[0x5401ef]
python3(PyEval_EvalFrameEx+0x50bf)[0x53b83f]
python3(PyEval_EvalFrameEx+0x4b14)[0x53b294]
python3[0x53fc97]
python3(PyEval_EvalCode+0x1f)[0x5409bf]
python3[0x60cb42]
python3(PyRun_FileExFlags+0x9a)[0x60efea]
python3(PyRun_SimpleFileExFlags+0x1bc)[0x60f7dc]
python3(Py_Main+0x456)[0x640256]
python3(main+0xe1)[0x4d0001]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0)[0x7fb3d9987830]
python3(_start+0x29)[0x5d6999]
======= Memory map: ========
00400000-007aa000 r-xp 00000000 103:02 1177926                           /usr/bin/python3.5
009a9000-009ab000 r--p 003a9000 103:02 1177926                           /usr/bin/python3.5
009ab000-00a42000 rw-p 003ab000 103:02 1177926                           /usr/bin/python3.5
00a42000-00a73000 rw-p 00000000 00:00 0
00afd000-02431000 rw-p 00000000 00:00 0                                  [heap]
7fb340000000-7fb340021000 rw-p 00000000 00:00 0
7fb340021000-7fb344000000 ---p 00000000 00:00 0
7fb348000000-7fb348021000 rw-p 00000000 00:00 0
7fb348021000-7fb34c000000 ---p 00000000 00:00 0
7fb34c000000-7fb34c021000 rw-p 00000000 00:00 0
7fb34c021000-7fb350000000 ---p 00000000 00:00 0
7fb350000000-7fb350021000 rw-p 00000000 00:00 0
7fb350021000-7fb354000000 ---p 00000000 00:00 0
7fb358000000-7fb358021000 rw-p 00000000 00:00 0
7fb358021000-7fb35c000000 ---p 00000000 00:00 0
7fb360000000-7fb360021000 rw-p 00000000 00:00 0
7fb360021000-7fb364000000 ---p 00000000 00:00 0
7fb364000000-7fb364021000 rw-p 00000000 00:00 0
7fb364021000-7fb368000000 ---p 00000000 00:00 0
7fb368000000-7fb368021000 rw-p 00000000 00:00 0
7fb368021000-7fb36c000000 ---p 00000000 00:00 0
7fb36c000000-7fb36c021000 rw-p 00000000 00:00 0
7fb36c021000-7fb370000000 ---p 00000000 00:00 0
7fb370000000-7fb370021000 rw-p 00000000 00:00 0
7fb370021000-7fb374000000 ---p 00000000 00:00 0
7fb374000000-7fb374021000 rw-p 00000000 00:00 0
7fb374021000-7fb378000000 ---p 00000000 00:00 0
7fb378000000-7fb378021000 rw-p 00000000 00:00 0
7fb378021000-7fb37c000000 ---p 00000000 00:00 0
7fb37e902000-7fb37e930000 r-xp 00000000 103:02 1177633                   /usr/lib/x86_64-linux-gnu/libgomp.so.1.0.0
7fb37e930000-7fb37eb30000 ---p 0002e000 103:02 1177633                   /usr/lib/x86_64-linux-gnu/libgomp.so.1.0.0
7fb37eb30000-7fb37eb31000 r--p 0002e000 103:02 1177633                   /usr/lib/x86_64-linux-gnu/libgomp.so.1.0.0
7fb37eb31000-7fb37eb32000 rw-p 0002f000 103:02 1177633                   /usr/lib/x86_64-linux-gnu/libgomp.so.1.0.0
7fb37eb32000-7fb381430000 r-xp 00000000 103:02 1413598                   /usr/local/lib/python3.5/dist-packages/xgboost/lib/libxgboost.so
7fb381430000-7fb38162f000 ---p 028fe000 103:02 1413598                   /usr/local/lib/python3.5/dist-packages/xgboost/lib/libxgboost.so
7fb38162f000-7fb381640000 rw-p 028fd000 103:02 1413598                   /usr/local/lib/python3.5/dist-packages/xgboost/lib/libxgboost.so
7fb381640000-7fb381684000 rw-p 00000000 00:00 0
7fb381684000-7fb381696000 r-xp 00000000 103:02 1217787                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/_random.cpython-35m-x86_64-linux-gnu.so
7fb381696000-7fb381896000 ---p 00012000 103:02 1217787                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/_random.cpython-35m-x86_64-linux-gnu.so
7fb381896000-7fb381899000 rw-p 00012000 103:02 1217787                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/_random.cpython-35m-x86_64-linux-gnu.so
7fb381899000-7fb3818ce000 r-xp 00000000 103:02 1217625                   /usr/local/lib/python3.5/dist-packages/sklearn/metrics/pairwise_fast.cpython-35m-x86_64-linux-gnu.so
7fb3818ce000-7fb381acd000 ---p 00035000 103:02 1217625                   /usr/local/lib/python3.5/dist-packages/sklearn/metrics/pairwise_fast.cpython-35m-x86_64-linux-gnu.so
7fb381acd000-7fb381ad1000 rw-p 00034000 103:02 1217625                   /usr/local/lib/python3.5/dist-packages/sklearn/metrics/pairwise_fast.cpython-35m-x86_64-linux-gnu.so
7fb381ad1000-7fb381b12000 rw-p 00000000 00:00 0
7fb381b12000-7fb381b16000 r-xp 00000000 103:02 1217788                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/lgamma.cpython-35m-x86_64-linux-gnu.so
7fb381b16000-7fb381d15000 ---p 00004000 103:02 1217788                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/lgamma.cpython-35m-x86_64-linux-gnu.so
7fb381d15000-7fb381d16000 rw-p 00003000 103:02 1217788                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/lgamma.cpython-35m-x86_64-linux-gnu.so
7fb381d16000-7fb381d2d000 r-xp 00000000 103:02 1217631                   /usr/local/lib/python3.5/dist-packages/sklearn/metrics/cluster/expected_mutual_info_fast.cpython-35m-x86_64-linux-gnu.so
7fb381d2d000-7fb381f2d000 ---p 00017000 103:02 1217631                   /usr/local/lib/python3.5/dist-packages/sklearn/metrics/cluster/expected_mutual_info_fast.cpython-35m-x86_64-linux-gnu.so
7fb381f2d000-7fb381f2f000 rw-p 00017000 103:02 1217631                   /usr/local/lib/python3.5/dist-packages/sklearn/metrics/cluster/expected_mutual_info_fast.cpython-35m-x86_64-linux-gnu.so
7fb381f2f000-7fb381f6f000 rw-p 00000000 00:00 0
7fb381f6f000-7fb381fd0000 r-xp 00000000 103:02 1217809                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/sparsefuncs_fast.cpython-35m-x86_64-linux-gnu.so
7fb381fd0000-7fb3821cf000 ---p 00061000 103:02 1217809                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/sparsefuncs_fast.cpython-35m-x86_64-linux-gnu.so
7fb3821cf000-7fb3821d5000 rw-p 00060000 103:02 1217809                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/sparsefuncs_fast.cpython-35m-x86_64-linux-gnu.so
7fb3821d5000-7fb3821d6000 rw-p 00000000 00:00 0
7fb3821d6000-7fb3821de000 r-xp 00000000 103:02 1217810                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/_logistic_sigmoid.cpython-35m-x86_64-linux-gnu.so
7fb3821de000-7fb3823de000 ---p 00008000 103:02 1217810                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/_logistic_sigmoid.cpython-35m-x86_64-linux-gnu.so
7fb3823de000-7fb3823df000 rw-p 00008000 103:02 1217810                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/_logistic_sigmoid.cpython-35m-x86_64-linux-gnu.so
7fb3823df000-7fb38241f000 rw-p 00000000 00:00 0
7fb38241f000-7fb382430000 r-xp 00000000 103:02 1488540                   /usr/local/lib/python3.5/dist-packages/scipy/stats/mvn.cpython-35m-x86_64-linux-gnu.so
7fb382430000-7fb38262f000 ---p 00011000 103:02 1488540                   /usr/local/lib/python3.5/dist-packages/scipy/stats/mvn.cpython-35m-x86_64-linux-gnu.so
7fb38262f000-7fb382631000 rw-p 00010000 103:02 1488540                   /usr/local/lib/python3.5/dist-packages/scipy/stats/mvn.cpython-35m-x86_64-linux-gnu.so
7fb382631000-7fb382728000 rw-p 00000000 00:00 0
7fb382728000-7fb38272a000 rw-p 00031000 103:02 1488540                   /usr/local/lib/python3.5/dist-packages/scipy/stats/mvn.cpython-35m-x86_64-linux-gnu.so
7fb38272a000-7fb38276a000 rw-p 00000000 00:00 0
7fb38276a000-7fb382774000 r-xp 00000000 103:02 1488548                   /usr/local/lib/python3.5/dist-packages/scipy/stats/statlib.cpython-35m-x86_64-linux-gnu.so
7fb382774000-7fb382973000 ---p 0000a000 103:02 1488548                   /usr/local/lib/python3.5/dist-packages/scipy/stats/statlib.cpython-35m-x86_64-linux-gnu.so
7fb382973000-7fb382975000 rw-p 00009000 103:02 1488548                   /usr/local/lib/python3.5/dist-packages/scipy/stats/statlib.cpython-35m-x86_64-linux-gnu.so
7fb382975000-7fb382977000 rw-p 00026000 103:02 1488548                   /usr/local/lib/python3.5/dist-packages/scipy/stats/statlib.cpython-35m-x86_64-linux-gnu.so
7fb382977000-7fb382a77000 rw-p 00000000 00:00 0
7fb382a77000-7fb382adb000 r-xp 00000000 103:02 1488541                   /usr/local/lib/python3.5/dist-packages/scipy/stats/_stats.cpython-35m-x86_64-linux-gnu.so
7fb382adb000-7fb382cdb000 ---p 00064000 103:02 1488541                   /usr/local/lib/python3.5/dist-packages/scipy/stats/_stats.cpython-35m-x86_64-linux-gnu.so
7fb382cdb000-7fb382ce0000 rw-p 00064000 103:02 1488541                   /usr/local/lib/python3.5/dist-packages/scipy/stats/_stats.cpython-35m-x86_64-linux-gnu.so
7fb382ce0000-7fb382da1000 rw-p 00000000 00:00 0
7fb382da1000-7fb382dba000 r-xp 00000000 103:02 1575791                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/lsoda.cpython-35m-x86_64-linux-gnu.so
7fb382dba000-7fb382fb9000 ---p 00019000 103:02 1575791                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/lsoda.cpython-35m-x86_64-linux-gnu.so
7fb382fb9000-7fb382fbb000 rw-p 00018000 103:02 1575791                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/lsoda.cpython-35m-x86_64-linux-gnu.so
7fb382fbb000-7fb382fbe000 rw-p 00048000 103:02 1575791                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/lsoda.cpython-35m-x86_64-linux-gnu.so
7fb382fbe000-7fb382fd9000 r-xp 00000000 103:02 1575780                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_dop.cpython-35m-x86_64-linux-gnu.so
7fb382fd9000-7fb3831d8000 ---p 0001b000 103:02 1575780                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_dop.cpython-35m-x86_64-linux-gnu.so
7fb3831d8000-7fb3831da000 rw-p 0001a000 103:02 1575780                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_dop.cpython-35m-x86_64-linux-gnu.so
7fb3831da000-7fb3831dc000 rw-p 00044000 103:02 1575780                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_dop.cpython-35m-x86_64-linux-gnu.so
7fb3831dc000-7fb38320d000 r-xp 00000000 103:02 1575785                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/vode.cpython-35m-x86_64-linux-gnu.so
7fb38320d000-7fb38340c000 ---p 00031000 103:02 1575785                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/vode.cpython-35m-x86_64-linux-gnu.so
7fb38340c000-7fb38340e000 rw-p 00030000 103:02 1575785                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/vode.cpython-35m-x86_64-linux-gnu.so
7fb38340e000-7fb38340f000 rw-p 00000000 00:00 0
7fb38340f000-7fb383412000 rw-p 00082000 103:02 1575785                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/vode.cpython-35m-x86_64-linux-gnu.so
7fb383412000-7fb38342c000 r-xp 00000000 103:02 1575788                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_quadpack.cpython-35m-x86_64-linux-gnu.so
7fb38342c000-7fb38362c000 ---p 0001a000 103:02 1575788                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_quadpack.cpython-35m-x86_64-linux-gnu.so
7fb38362c000-7fb38362d000 rw-p 0001a000 103:02 1575788                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_quadpack.cpython-35m-x86_64-linux-gnu.so
7fb38362d000-7fb383630000 rw-p 00054000 103:02 1575788                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_quadpack.cpython-35m-x86_64-linux-gnu.so
7fb383630000-7fb383646000 r-xp 00000000 103:02 1575792                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_odepack.cpython-35m-x86_64-linux-gnu.so
7fb383646000-7fb383845000 ---p 00016000 103:02 1575792                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_odepack.cpython-35m-x86_64-linux-gnu.so
7fb383845000-7fb383846000 rw-p 00015000 103:02 1575792                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_odepack.cpython-35m-x86_64-linux-gnu.so
7fb383846000-7fb383847000 rw-p 00000000 00:00 0
7fb383847000-7fb38384a000 rw-p 0003a000 103:02 1575792                   /usr/local/lib/python3.5/dist-packages/scipy/integrate/_odepack.cpython-35m-x86_64-linux-gnu.so
7fb38384a000-7fb38388a000 rw-p 00000000 00:00 0
7fb38388a000-7fb383894000 r-xp 00000000 103:02 1488175                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_nnls.cpython-35m-x86_64-linux-gnu.so
7fb383894000-7fb383a94000 ---p 0000a000 103:02 1488175                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_nnls.cpython-35m-x86_64-linux-gnu.so
7fb383a94000-7fb383a95000 rw-p 0000a000 103:02 1488175                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_nnls.cpython-35m-x86_64-linux-gnu.so
7fb383a95000-7fb383a97000 rw-p 00022000 103:02 1488175                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_nnls.cpython-35m-x86_64-linux-gnu.so
7fb383a97000-7fb383a9a000 r-xp 00000000 103:02 1488184                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_zeros.cpython-35m-x86_64-linux-gnu.so
7fb383a9a000-7fb383c99000 ---p 00003000 103:02 1488184                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_zeros.cpython-35m-x86_64-linux-gnu.so
7fb383c99000-7fb383c9a000 rw-p 00002000 103:02 1488184                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_zeros.cpython-35m-x86_64-linux-gnu.so
7fb383c9a000-7fb383cc1000 r-xp 00000000 103:02 1488227                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_lsq/givens_elimination.cpython-35m-x86_64-linux-gnu.so
7fb383cc1000-7fb383ec0000 ---p 00027000 103:02 1488227                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_lsq/givens_elimination.cpython-35m-x86_64-linux-gnu.so
7fb383ec0000-7fb383ec4000 rw-p 00026000 103:02 1488227                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_lsq/givens_elimination.cpython-35m-x86_64-linux-gnu.so
7fb383ec4000-7fb383f04000 rw-p 00000000 00:00 0
7fb383f04000-7fb383f23000 r-xp 00000000 103:02 1488164                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_minpack.cpython-35m-x86_64-linux-gnu.so
7fb383f23000-7fb384122000 ---p 0001f000 103:02 1488164                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_minpack.cpython-35m-x86_64-linux-gnu.so
7fb384122000-7fb384123000 rw-p 0001e000 103:02 1488164                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_minpack.cpython-35m-x86_64-linux-gnu.so
7fb384123000-7fb384125000 rw-p 0004e000 103:02 1488164                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_minpack.cpython-35m-x86_64-linux-gnu.so
7fb384125000-7fb38413b000 r-xp 00000000 103:02 1488186                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_slsqp.cpython-35m-x86_64-linux-gnu.so
7fb38413b000-7fb38433b000 ---p 00016000 103:02 1488186                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_slsqp.cpython-35m-x86_64-linux-gnu.so
7fb38433b000-7fb38433c000 rw-p 00016000 103:02 1488186                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_slsqp.cpython-35m-x86_64-linux-gnu.so
7fb38433c000-7fb38433e000 rw-p 00040000 103:02 1488186                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_slsqp.cpython-35m-x86_64-linux-gnu.so
7fb38433e000-7fb38435b000 r-xp 00000000 103:02 1488161                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_cobyla.cpython-35m-x86_64-linux-gnu.so
7fb38435b000-7fb38455b000 ---p 0001d000 103:02 1488161                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_cobyla.cpython-35m-x86_64-linux-gnu.so
7fb38455b000-7fb38455c000 rw-p 0001d000 103:02 1488161                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_cobyla.cpython-35m-x86_64-linux-gnu.so
7fb38455c000-7fb38455e000 rw-p 00046000 103:02 1488161                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_cobyla.cpython-35m-x86_64-linux-gnu.so
7fb38455e000-7fb38456a000 r-xp 00000000 103:02 1488173                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/moduleTNC.cpython-35m-x86_64-linux-gnu.so
7fb38456a000-7fb384769000 ---p 0000c000 103:02 1488173                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/moduleTNC.cpython-35m-x86_64-linux-gnu.so
7fb384769000-7fb38476a000 rw-p 0000b000 103:02 1488173                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/moduleTNC.cpython-35m-x86_64-linux-gnu.so
7fb38476a000-7fb384787000 r-xp 00000000 103:02 1488169                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_lbfgsb.cpython-35m-x86_64-linux-gnu.so
7fb384787000-7fb384987000 ---p 0001d000 103:02 1488169                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_lbfgsb.cpython-35m-x86_64-linux-gnu.so
7fb384987000-7fb384988000 rw-p 0001d000 103:02 1488169                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_lbfgsb.cpython-35m-x86_64-linux-gnu.so
7fb384988000-7fb38498b000 rw-p 00051000 103:02 1488169                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_lbfgsb.cpython-35m-x86_64-linux-gnu.so
7fb38498b000-7fb3849cb000 rw-p 00000000 00:00 0
7fb3849cb000-7fb3849f6000 r-xp 00000000 103:02 1488165                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_group_columns.cpython-35m-x86_64-linux-gnu.so
7fb3849f6000-7fb384bf5000 ---p 0002b000 103:02 1488165                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_group_columns.cpython-35m-x86_64-linux-gnu.so
7fb384bf5000-7fb384bf9000 rw-p 0002a000 103:02 1488165                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_group_columns.cpython-35m-x86_64-linux-gnu.so
7fb384bf9000-7fb384c42000 r-xp 00000000 103:02 1488344                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_trlib/_trlib.cpython-35m-x86_64-linux-gnu.so
7fb384c42000-7fb384e42000 ---p 00049000 103:02 1488344                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_trlib/_trlib.cpython-35m-x86_64-linux-gnu.so
7fb384e42000-7fb384e46000 rw-p 00049000 103:02 1488344                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_trlib/_trlib.cpython-35m-x86_64-linux-gnu.so
7fb384e46000-7fb384e47000 rw-p 00000000 00:00 0
7fb384e47000-7fb384e4b000 rw-p 0018f000 103:02 1488344                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/_trlib/_trlib.cpython-35m-x86_64-linux-gnu.so
7fb384e4b000-7fb384e54000 r-xp 00000000 103:02 1488192                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/minpack2.cpython-35m-x86_64-linux-gnu.so
7fb384e54000-7fb385054000 ---p 00009000 103:02 1488192                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/minpack2.cpython-35m-x86_64-linux-gnu.so
7fb385054000-7fb385055000 rw-p 00009000 103:02 1488192                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/minpack2.cpython-35m-x86_64-linux-gnu.so
7fb385055000-7fb385057000 rw-p 00022000 103:02 1488192                   /usr/local/lib/python3.5/dist-packages/scipy/optimize/minpack2.cpython-35m-x86_64-linux-gnu.so
7fb385057000-7fb385097000 rw-p 00000000 00:00 0
7fb385097000-7fb3850ca000 r-xp 00000000 103:02 1851293                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/liblzma-90de1f11.so.5.2.2
7fb3850ca000-7fb3852c9000 ---p 00033000 103:02 1851293                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/liblzma-90de1f11.so.5.2.2
7fb3852c9000-7fb3852ca000 rw-p 00032000 103:02 1851293                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/liblzma-90de1f11.so.5.2.2
7fb3852ca000-7fb3852cb000 rw-p 00034000 103:02 1851293                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/liblzma-90de1f11.so.5.2.2
7fb3852cb000-7fb3852d2000 r-xp 00000000 103:02 533998                    /lib/x86_64-linux-gnu/librt-2.23.so
7fb3852d2000-7fb3854d1000 ---p 00007000 103:02 533998                    /lib/x86_64-linux-gnu/librt-2.23.so
7fb3854d1000-7fb3854d2000 r--p 00006000 103:02 533998                    /lib/x86_64-linux-gnu/librt-2.23.so
7fb3854d2000-7fb3854d3000 rw-p 00007000 103:02 533998                    /lib/x86_64-linux-gnu/librt-2.23.so
7fb3854d3000-7fb385562000 r-xp 00000000 103:02 1851295                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libtiff-8a6d997d.so.5.3.0
7fb385562000-7fb385762000 ---p 0008f000 103:02 1851295                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libtiff-8a6d997d.so.5.3.0
7fb385762000-7fb385766000 rw-p 0008f000 103:02 1851295                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libtiff-8a6d997d.so.5.3.0
7fb385766000-7fb38576e000 rw-p 00094000 103:02 1851295                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libtiff-8a6d997d.so.5.3.0
7fb38576e000-7fb385782000 r-xp 00000000 103:02 1851296                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libz-a147dcb0.so.1.2.3
7fb385782000-7fb385981000 ---p 00014000 103:02 1851296                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libz-a147dcb0.so.1.2.3
7fb385981000-7fb385982000 rw-p 00013000 103:02 1851296                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libz-a147dcb0.so.1.2.3
7fb385982000-7fb385983000 rw-p 00015000 103:02 1851296                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libz-a147dcb0.so.1.2.3
7fb385983000-7fb3859ca000 r-xp 00000000 103:02 1851291                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libopenjp2-e366d6b0.so.2.1.0
7fb3859ca000-7fb385bca000 ---p 00047000 103:02 1851291                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libopenjp2-e366d6b0.so.2.1.0
7fb385bca000-7fb385bcc000 rw-p 00047000 103:02 1851291                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libopenjp2-e366d6b0.so.2.1.0
7fb385bcc000-7fb385bce000 rw-p 0004a000 103:02 1851291                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libopenjp2-e366d6b0.so.2.1.0
7fb385bce000-7fb385c1f000 r-xp 00000000 103:02 1851292                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libjpeg-3fe7dfc0.so.9.3.0
7fb385c1f000-7fb385e1f000 ---p 00051000 103:02 1851292                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libjpeg-3fe7dfc0.so.9.3.0
7fb385e1f000-7fb385e20000 rw-p 00051000 103:02 1851292                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libjpeg-3fe7dfc0.so.9.3.0
7fb385e20000-7fb385e22000 rw-p 00053000 103:02 1851292                   /usr/local/lib/python3.5/dist-packages/PIL/.libs/libjpeg-3fe7dfc0.so.9.3.0
7fb385e22000-7fb385e9c000 r-xp 00000000 103:02 1733338                   /usr/local/lib/python3.5/dist-packages/PIL/_imaging.cpython-35m-x86_64-linux-gnu.so
7fb385e9c000-7fb38609c000 ---p 0007a000 103:02 1733338                   /usr/local/lib/python3.5/dist-packages/PIL/_imaging.cpython-35m-x86_64-linux-gnu.so
7fb38609c000-7fb3860b2000 rw-p 0007a000 103:02 1733338                   /usr/local/lib/python3.5/dist-packages/PIL/_imaging.cpython-35m-x86_64-linux-gnu.so
7fb3860b2000-7fb3860f2000 rw-p 00000000 00:00 0
7fb3860f2000-7fb38611c000 r-xp 00000000 103:02 1488378                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_hausdorff.cpython-35m-x86_64-linux-gnu.so
7fb38611c000-7fb38631b000 ---p 0002a000 103:02 1488378                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_hausdorff.cpython-35m-x86_64-linux-gnu.so
7fb38631b000-7fb38631f000 rw-p 00029000 103:02 1488378                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_hausdorff.cpython-35m-x86_64-linux-gnu.so
7fb38631f000-7fb386334000 r-xp 00000000 103:02 1488379                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_distance_wrap.cpython-35m-x86_64-linux-gnu.so
7fb386334000-7fb386534000 ---p 00015000 103:02 1488379                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_distance_wrap.cpython-35m-x86_64-linux-gnu.so
7fb386534000-7fb386535000 rw-p 00015000 103:02 1488379                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_distance_wrap.cpython-35m-x86_64-linux-gnu.so
7fb386535000-7fb38655e000 r-xp 00000000 103:02 1488372                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_voronoi.cpython-35m-x86_64-linux-gnu.so
7fb38655e000-7fb38675e000 ---p 00029000 103:02 1488372                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_voronoi.cpython-35m-x86_64-linux-gnu.so
7fb38675e000-7fb386761000 rw-p 00029000 103:02 1488372                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/_voronoi.cpython-35m-x86_64-linux-gnu.so
7fb386761000-7fb3867a2000 rw-p 00000000 00:00 0
7fb3867a2000-7fb3867ab000 r-xp 00000000 103:02 1575907                   /usr/local/lib/python3.5/dist-packages/scipy/_lib/messagestream.cpython-35m-x86_64-linux-gnu.so
7fb3867ab000-7fb3869ab000 ---p 00009000 103:02 1575907                   /usr/local/lib/python3.5/dist-packages/scipy/_lib/messagestream.cpython-35m-x86_64-linux-gnu.so
7fb3869ab000-7fb3869ac000 rw-p 00009000 103:02 1575907                   /usr/local/lib/python3.5/dist-packages/scipy/_lib/messagestream.cpython-35m-x86_64-linux-gnu.so
7fb3869ac000-7fb386a7e000 r-xp 00000000 103:02 1488374                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/qhull.cpython-35m-x86_64-linux-gnu.so
7fb386a7e000-7fb386c7d000 ---p 000d2000 103:02 1488374                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/qhull.cpython-35m-x86_64-linux-gnu.so
7fb386c7d000-7fb386c86000 rw-p 000d1000 103:02 1488374                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/qhull.cpython-35m-x86_64-linux-gnu.so
7fb386c86000-7fb386c88000 rw-p 00000000 00:00 0
7fb386c88000-7fb386c8e000 rw-p 003e1000 103:02 1488374                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/qhull.cpython-35m-x86_64-linux-gnu.so
7fb386c8e000-7fb386d23000 r-xp 00000000 103:02 1488370                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/ckdtree.cpython-35m-x86_64-linux-gnu.so
7fb386d23000-7fb386f22000 ---p 00095000 103:02 1488370                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/ckdtree.cpython-35m-x86_64-linux-gnu.so
7fb386f22000-7fb386f2b000 rw-p 00094000 103:02 1488370                   /usr/local/lib/python3.5/dist-packages/scipy/spatial/ckdtree.cpython-35m-x86_64-linux-gnu.so
7fb386f2b000-7fb386f7c000 r-xp 00000000 103:02 1488460                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/interpnd.cpython-35m-x86_64-linux-gnu.so
7fb386f7c000-7fb38717c000 ---p 00051000 103:02 1488460                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/interpnd.cpython-35m-x86_64-linux-gnu.so
7fb38717c000-7fb387181000 rw-p 00051000 103:02 1488460                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/interpnd.cpython-35m-x86_64-linux-gnu.so
7fb387181000-7fb387182000 rw-p 00000000 00:00 0
7fb387182000-7fb3871d0000 r-xp 00000000 103:02 1488442                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_ppoly.cpython-35m-x86_64-linux-gnu.so
7fb3871d0000-7fb3873d0000 ---p 0004e000 103:02 1488442                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_ppoly.cpython-35m-x86_64-linux-gnu.so
7fb3873d0000-7fb3873d6000 rw-p 0004e000 103:02 1488442                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_ppoly.cpython-35m-x86_64-linux-gnu.so
7fb3873d6000-7fb3873d7000 rw-p 00000000 00:00 0
7fb3873d7000-7fb3873db000 rw-p 0019f000 103:02 1488442                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_ppoly.cpython-35m-x86_64-linux-gnu.so
7fb3873db000-7fb387417000 r-xp 00000000 103:02 1488456                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_bspl.cpython-35m-x86_64-linux-gnu.so
7fb387417000-7fb387616000 ---p 0003c000 103:02 1488456                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_bspl.cpython-35m-x86_64-linux-gnu.so
7fb387616000-7fb38761c000 rw-p 0003b000 103:02 1488456                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_bspl.cpython-35m-x86_64-linux-gnu.so
7fb38761c000-7fb38761d000 rw-p 00000000 00:00 0
7fb38761d000-7fb387620000 rw-p 0013f000 103:02 1488456                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_bspl.cpython-35m-x86_64-linux-gnu.so
7fb387620000-7fb387660000 rw-p 00000000 00:00 0
7fb387660000-7fb3876bf000 r-xp 00000000 103:02 1488450                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/dfitpack.cpython-35m-x86_64-linux-gnu.so
7fb3876bf000-7fb3878bf000 ---p 0005f000 103:02 1488450                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/dfitpack.cpython-35m-x86_64-linux-gnu.so
7fb3878bf000-7fb3878c6000 rw-p 0005f000 103:02 1488450                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/dfitpack.cpython-35m-x86_64-linux-gnu.so
7fb3878c6000-7fb3878c8000 rw-p 00112000 103:02 1488450                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/dfitpack.cpython-35m-x86_64-linux-gnu.so
7fb3878c8000-7fb3878fc000 r-xp 00000000 103:02 1488453                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_fitpack.cpython-35m-x86_64-linux-gnu.so
7fb3878fc000-7fb387afc000 ---p 00034000 103:02 1488453                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_fitpack.cpython-35m-x86_64-linux-gnu.so
7fb387afc000-7fb387afd000 rw-p 00034000 103:02 1488453                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_fitpack.cpython-35m-x86_64-linux-gnu.so
7fb387afd000-7fb387aff000 rw-p 00091000 103:02 1488453                   /usr/local/lib/python3.5/dist-packages/scipy/interpolate/_fitpack.cpython-35m-x86_64-linux-gnu.so
7fb387aff000-7fb387bbf000 rw-p 00000000 00:00 0
7fb387bbf000-7fb387bc2000 r-xp 00000000 103:02 1179424                   /usr/lib/python3.5/lib-dynload/_multiprocessing.cpython-35m-x86_64-linux-gnu.so
7fb387bc2000-7fb387dc1000 ---p 00003000 103:02 1179424                   /usr/lib/python3.5/lib-dynload/_multiprocessing.cpython-35m-x86_64-linux-gnu.so
7fb387dc1000-7fb387dc2000 r--p 00002000 103:02 1179424                   /usr/lib/python3.5/lib-dynload/_multiprocessing.cpython-35m-x86_64-linux-gnu.so
7fb387dc2000-7fb387dc3000 rw-p 00003000 103:02 1179424                   /usr/lib/python3.5/lib-dynload/_multiprocessing.cpython-35m-x86_64-linux-gnu.so
7fb387dc3000-7fb387e43000 rw-p 00000000 00:00 0
7fb387e43000-7fb387e54000 r-xp 00000000 103:02 1336266                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ellip_harm_2.cpython-35m-x86_64-linux-gnu.so
7fb387e54000-7fb388054000 ---p 00011000 103:02 1336266                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ellip_harm_2.cpython-35m-x86_64-linux-gnu.so
7fb388054000-7fb388056000 rw-p 00011000 103:02 1336266                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ellip_harm_2.cpython-35m-x86_64-linux-gnu.so
7fb388056000-7fb38805a000 rw-p 0005a000 103:02 1336266                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ellip_harm_2.cpython-35m-x86_64-linux-gnu.so
7fb38805a000-7fb388060000 r-xp 00000000 103:02 1336265                   /usr/local/lib/python3.5/dist-packages/scipy/special/_comb.cpython-35m-x86_64-linux-gnu.so
7fb388060000-7fb388260000 ---p 00006000 103:02 1336265                   /usr/local/lib/python3.5/dist-packages/scipy/special/_comb.cpython-35m-x86_64-linux-gnu.so
7fb388260000-7fb388261000 rw-p 00006000 103:02 1336265                   /usr/local/lib/python3.5/dist-packages/scipy/special/_comb.cpython-35m-x86_64-linux-gnu.so
7fb388261000-7fb388316000 r-xp 00000000 103:02 1336260                   /usr/local/lib/python3.5/dist-packages/scipy/special/specfun.cpython-35m-x86_64-linux-gnu.so
7fb388316000-7fb388516000 ---p 000b5000 103:02 1336260                   /usr/local/lib/python3.5/dist-packages/scipy/special/specfun.cpython-35m-x86_64-linux-gnu.so
7fb388516000-7fb38851d000 rw-p 000b5000 103:02 1336260                   /usr/local/lib/python3.5/dist-packages/scipy/special/specfun.cpython-35m-x86_64-linux-gnu.so
7fb38851d000-7fb38851f000 rw-p 0019d000 103:02 1336260                   /usr/local/lib/python3.5/dist-packages/scipy/special/specfun.cpython-35m-x86_64-linux-gnu.so
7fb38851f000-7fb38855f000 rw-p 00000000 00:00 0
7fb38855f000-7fb38857b000 r-xp 00000000 103:02 1336269                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ufuncs_cxx.cpython-35m-x86_64-linux-gnu.so
7fb38857b000-7fb38877b000 ---p 0001c000 103:02 1336269                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ufuncs_cxx.cpython-35m-x86_64-linux-gnu.so
7fb38877b000-7fb38877c000 rw-p 0001c000 103:02 1336269                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ufuncs_cxx.cpython-35m-x86_64-linux-gnu.so
7fb38877c000-7fb38877d000 rw-p 00000000 00:00 0
7fb38877d000-7fb3888fc000 r-xp 00000000 103:02 1336270                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ufuncs.cpython-35m-x86_64-linux-gnu.so
7fb3888fc000-7fb388afb000 ---p 0017f000 103:02 1336270                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ufuncs.cpython-35m-x86_64-linux-gnu.so
7fb388afb000-7fb388b04000 rw-p 0017e000 103:02 1336270                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ufuncs.cpython-35m-x86_64-linux-gnu.so
7fb388b04000-7fb388b0c000 rw-p 00000000 00:00 0
7fb388b0c000-7fb388b11000 rw-p 004b5000 103:02 1336270                   /usr/local/lib/python3.5/dist-packages/scipy/special/_ufuncs.cpython-35m-x86_64-linux-gnu.so
7fb388b11000-7fb388b9a000 r-xp 00000000 103:02 1575564                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/eigen/arpack/_arpack.cpython-35m-x86_64-linux-gnu.so
7fb388b9a000-7fb388d9a000 ---p 00089000 103:02 1575564                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/eigen/arpack/_arpack.cpython-35m-x86_64-linux-gnu.so
7fb388d9a000-7fb388da5000 rw-p 00089000 103:02 1575564                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/eigen/arpack/_arpack.cpython-35m-x86_64-linux-gnu.so
7fb388da5000-7fb388da7000 rw-p 00000000 00:00 0
7fb388da7000-7fb388dab000 rw-p 0017c000 103:02 1575564                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/eigen/arpack/_arpack.cpython-35m-x86_64-linux-gnu.so
7fb388dab000-7fb388df8000 r-xp 00000000 103:02 1575600                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/dsolve/_superlu.cpython-35m-x86_64-linux-gnu.so
7fb388df8000-7fb388ff7000 ---p 0004d000 103:02 1575600                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/dsolve/_superlu.cpython-35m-x86_64-linux-gnu.so
7fb388ff7000-7fb388ff9000 rw-p 0004c000 103:02 1575600                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/dsolve/_superlu.cpython-35m-x86_64-linux-gnu.so
7fb388ff9000-7fb388ffd000 rw-p 00196000 103:02 1575600                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/dsolve/_superlu.cpython-35m-x86_64-linux-gnu.so
7fb388ffd000-7fb38903d000 rw-p 00000000 00:00 0
7fb38903d000-7fb3890c8000 r-xp 00000000 103:02 1336142                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_lapack.cpython-35m-x86_64-linux-gnu.so
7fb3890c8000-7fb3892c7000 ---p 0008b000 103:02 1336142                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_lapack.cpython-35m-x86_64-linux-gnu.so
7fb3892c7000-7fb3892cb000 rw-p 0008a000 103:02 1336142                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_lapack.cpython-35m-x86_64-linux-gnu.so
7fb3892cb000-7fb3892dc000 rw-p 001eb000 103:02 1336142                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_lapack.cpython-35m-x86_64-linux-gnu.so
7fb3892dc000-7fb389316000 r-xp 00000000 103:02 1336133                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_blas.cpython-35m-x86_64-linux-gnu.so
7fb389316000-7fb389516000 ---p 0003a000 103:02 1336133                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_blas.cpython-35m-x86_64-linux-gnu.so
7fb389516000-7fb38951a000 rw-p 0003a000 103:02 1336133                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_blas.cpython-35m-x86_64-linux-gnu.so
7fb38951a000-7fb389521000 rw-p 00133000 103:02 1336133                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/cython_blas.cpython-35m-x86_64-linux-gnu.so
7fb389521000-7fb389569000 r-xp 00000000 103:02 1336134                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_decomp_update.cpython-35m-x86_64-linux-gnu.so
7fb389569000-7fb389768000 ---p 00048000 103:02 1336134                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_decomp_update.cpython-35m-x86_64-linux-gnu.so
7fb389768000-7fb38976e000 rw-p 00047000 103:02 1336134                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_decomp_update.cpython-35m-x86_64-linux-gnu.so
7fb38976e000-7fb3897af000 rw-p 00000000 00:00 0
7fb3897af000-7fb3897e3000 r-xp 00000000 103:02 1336104                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_solve_toeplitz.cpython-35m-x86_64-linux-gnu.so
7fb3897e3000-7fb3899e3000 ---p 00034000 103:02 1336104                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_solve_toeplitz.cpython-35m-x86_64-linux-gnu.so
7fb3899e3000-7fb3899e7000 rw-p 00034000 103:02 1336104                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_solve_toeplitz.cpython-35m-x86_64-linux-gnu.so
7fb3899e7000-7fb3899e8000 rw-p 00000000 00:00 0
7fb3899e8000-7fb3899f5000 r-xp 00000000 103:02 1336108                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flinalg.cpython-35m-x86_64-linux-gnu.so
7fb3899f5000-7fb389bf5000 ---p 0000d000 103:02 1336108                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flinalg.cpython-35m-x86_64-linux-gnu.so
7fb389bf5000-7fb389bf8000 rw-p 0000d000 103:02 1336108                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flinalg.cpython-35m-x86_64-linux-gnu.so
7fb389bf8000-7fb389bfb000 rw-p 00032000 103:02 1336108                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flinalg.cpython-35m-x86_64-linux-gnu.so
7fb389bfb000-7fb389ce2000 r-xp 00000000 103:02 1336129                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flapack.cpython-35m-x86_64-linux-gnu.so
7fb389ce2000-7fb389ee2000 ---p 000e7000 103:02 1336129                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flapack.cpython-35m-x86_64-linux-gnu.so
7fb389ee2000-7fb389f37000 rw-p 000e7000 103:02 1336129                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flapack.cpython-35m-x86_64-linux-gnu.so
7fb389f37000-7fb389f3f000 rw-p 00335000 103:02 1336129                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_flapack.cpython-35m-x86_64-linux-gnu.so
7fb389f3f000-7fb389fac000 r-xp 00000000 103:02 1336111                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_fblas.cpython-35m-x86_64-linux-gnu.so
7fb389fac000-7fb38a1ac000 ---p 0006d000 103:02 1336111                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_fblas.cpython-35m-x86_64-linux-gnu.so
7fb38a1ac000-7fb38a1d2000 rw-p 0006d000 103:02 1336111                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_fblas.cpython-35m-x86_64-linux-gnu.so
7fb38a1d2000-7fb38a1d7000 rw-p 00158000 103:02 1336111                   /usr/local/lib/python3.5/dist-packages/scipy/linalg/_fblas.cpython-35m-x86_64-linux-gnu.so
7fb38a1d7000-7fb38a217000 rw-p 00000000 00:00 0
7fb38a217000-7fb38a249000 r-xp 00000000 103:02 1575500                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/isolve/_iterative.cpython-35m-x86_64-linux-gnu.so
7fb38a249000-7fb38a449000 ---p 00032000 103:02 1575500                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/isolve/_iterative.cpython-35m-x86_64-linux-gnu.so
7fb38a449000-7fb38a450000 rw-p 00032000 103:02 1575500                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/isolve/_iterative.cpython-35m-x86_64-linux-gnu.so
7fb38a450000-7fb38a452000 rw-p 00000000 00:00 0
7fb38a452000-7fb38a456000 rw-p 000a5000 103:02 1575500                   /usr/local/lib/python3.5/dist-packages/scipy/sparse/linalg/isolve/_iterative.cpython-35m-x86_64-linux-gnu.so
7fb38a456000-7fb38a468000 r-xp 00000000 103:02 1217804                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/murmurhash.cpython-35m-x86_64-linux-gnu.so
7fb38a468000-7fb38a667000 ---p 00012000 103:02 1217804                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/murmurhash.cpython-35m-x86_64-linux-gnu.so
7fb38a667000-7fb38a669000 rw-p 00011000 103:02 1217804                   /usr/local/lib/python3.5/dist-packages/sklearn/utils/murmurhash.cpython-35m-x86_64-linux-gnu.so
7fb38a669000-7fb38a6a9000 rw-p 00000000 00:00 0
7fb38a6a9000-7fb38a6ac000 r-xp 00000000 103:02 1217400                   /usr/local/lib/python3.5/dist-packages/sklearn/__check_build/_check_build.cpython-35m-x86_64-linux-gnu.so
7fb38a6ac000-7fb38a8ab000 ---p 00003000 103:02 1217400                   /usr/local/lib/python3.5/dist-packages/sklearn/__check_build/_check_build.cpython-35m-x86_64-linux-gnu.so
7fb38a8ab000-7fb38a8ac000 rw-p 00002000 103:02 1217400                   /usr/local/lib/python3.5/dist-packages/sklearn/__check_build/_check_build.cpython-35m-x86_64-linux-gnu.so
7fb38a8ac000-7fb38a8be000 r-xp 00000000 103:02 1981462                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/testing.cpython-35m-x86_64-linux-gnu.so
7fb38a8be000-7fb38aabd000 ---p 00012000 103:02 1981462                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/testing.cpython-35m-x86_64-linux-gnu.so
7fb38aabd000-7fb38aabf000 rw-p 00011000 103:02 1981462                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/testing.cpython-35m-x86_64-linux-gnu.so
7fb38aabf000-7fb38aac0000 r-xp 00000000 103:02 1981521                   /usr/local/lib/python3.5/dist-packages/pandas/util/_move.cpython-35m-x86_64-linux-gnu.so
7fb38aac0000-7fb38acc0000 ---p 00001000 103:02 1981521                   /usr/local/lib/python3.5/dist-packages/pandas/util/_move.cpython-35m-x86_64-linux-gnu.so
7fb38acc0000-7fb38acc1000 rw-p 00001000 103:02 1981521                   /usr/local/lib/python3.5/dist-packages/pandas/util/_move.cpython-35m-x86_64-linux-gnu.so
7fb38acc1000-7fb38acd7000 r-xp 00000000 103:02 1975266                   /usr/local/lib/python3.5/dist-packages/pandas/io/msgpack/_unpacker.cpython-35m-x86_64-linux-gnu.so
7fb38acd7000-7fb38aed6000 ---p 00016000 103:02 1975266                   /usr/local/lib/python3.5/dist-packages/pandas/io/msgpack/_unpacker.cpython-35m-x86_64-linux-gnu.so
7fb38aed6000-7fb38aed9000 rw-p 00015000 103:02 1975266                   /usr/local/lib/python3.5/dist-packages/pandas/io/msgpack/_unpacker.cpython-35m-x86_64-linux-gnu.so
7fb38aed9000-7fb38aeea000 r-xp 00000000 103:02 1975268                   /usr/local/lib/python3.5/dist-packages/pandas/io/msgpack/_packer.cpython-35m-x86_64-linux-gnu.so
7fb38aeea000-7fb38b0e9000 ---p 00011000 103:02 1975268                   /usr/local/lib/python3.5/dist-packages/pandas/io/msgpack/_packer.cpython-35m-x86_64-linux-gnu.so
7fb38b0e9000-7fb38b0eb000 rw-p 00010000 103:02 1975268                   /usr/local/lib/python3.5/dist-packages/pandas/io/msgpack/_packer.cpython-35m-x86_64-linux-gnu.so
7fb38b0eb000-7fb38b1ab000 rw-p 00000000 00:00 0
7fb38b1ab000-7fb38b1de000 r-xp 00000000 103:02 1981470                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/writers.cpython-35m-x86_64-linux-gnu.so
7fb38b1de000-7fb38b3de000 ---p 00033000 103:02 1981470                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/writers.cpython-35m-x86_64-linux-gnu.so
7fb38b3de000-7fb38b3e2000 rw-p 00033000 103:02 1981470                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/writers.cpython-35m-x86_64-linux-gnu.so
7fb38b3e2000-7fb38b463000 rw-p 00000000 00:00 0
7fb38b463000-7fb38b478000 r-xp 00000000 103:02 1981476                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/json.cpython-35m-x86_64-linux-gnu.so
7fb38b478000-7fb38b678000 ---p 00015000 103:02 1981476                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/json.cpython-35m-x86_64-linux-gnu.so
7fb38b678000-7fb38b679000 rw-p 00015000 103:02 1981476                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/json.cpython-35m-x86_64-linux-gnu.so
7fb38b679000-7fb38b6fd000 r-xp 00000000 103:02 1981477                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/parsers.cpython-35m-x86_64-linux-gnu.so
7fb38b6fd000-7fb38b8fc000 ---p 00084000 103:02 1981477                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/parsers.cpython-35m-x86_64-linux-gnu.so
7fb38b8fc000-7fb38b903000 rw-p 00083000 103:02 1981477                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/parsers.cpython-35m-x86_64-linux-gnu.so
7fb38b903000-7fb38b985000 rw-p 00000000 00:00 0
7fb38b985000-7fb38b99d000 r-xp 00000000 103:02 1981472                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/reshape.cpython-35m-x86_64-linux-gnu.so
7fb38b99d000-7fb38bb9d000 ---p 00018000 103:02 1981472                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/reshape.cpython-35m-x86_64-linux-gnu.so
7fb38bb9d000-7fb38bba0000 rw-p 00018000 103:02 1981472                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/reshape.cpython-35m-x86_64-linux-gnu.so
7fb38bba0000-7fb38bbe0000 rw-p 00000000 00:00 0
7fb38bbe0000-7fb38bc8d000 r-xp 00000000 103:02 1981473                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/groupby.cpython-35m-x86_64-linux-gnu.so
7fb38bc8d000-7fb38be8c000 ---p 000ad000 103:02 1981473                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/groupby.cpython-35m-x86_64-linux-gnu.so
7fb38be8c000-7fb38be95000 rw-p 000ac000 103:02 1981473                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/groupby.cpython-35m-x86_64-linux-gnu.so
7fb38be95000-7fb38be96000 rw-p 00000000 00:00 0
7fb38be96000-7fb38bed5000 r-xp 00000000 103:02 1981465                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/reduction.cpython-35m-x86_64-linux-gnu.so
7fb38bed5000-7fb38c0d5000 ---p 0003f000 103:02 1981465                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/reduction.cpython-35m-x86_64-linux-gnu.so
7fb38c0d5000-7fb38c0d9000 rw-p 0003f000 103:02 1981465                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/reduction.cpython-35m-x86_64-linux-gnu.so
7fb38c0d9000-7fb38c119000 rw-p 00000000 00:00 0
7fb38c119000-7fb38c12e000 r-xp 00000000 103:02 1981482                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/skiplist.cpython-35m-x86_64-linux-gnu.so
7fb38c12e000-7fb38c32e000 ---p 00015000 103:02 1981482                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/skiplist.cpython-35m-x86_64-linux-gnu.so
7fb38c32e000-7fb38c330000 rw-p 00015000 103:02 1981482                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/skiplist.cpython-35m-x86_64-linux-gnu.so
7fb38c330000-7fb38c3d5000 r-xp 00000000 103:02 1981474                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/window.cpython-35m-x86_64-linux-gnu.so
7fb38c3d5000-7fb38c5d5000 ---p 000a5000 103:02 1981474                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/window.cpython-35m-x86_64-linux-gnu.so
7fb38c5d5000-7fb38c5dd000 rw-p 000a5000 103:02 1981474                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/window.cpython-35m-x86_64-linux-gnu.so
7fb38c5dd000-7fb38c65e000 rw-p 00000000 00:00 0
7fb38c65e000-7fb38c68c000 r-xp 00000000 103:02 1351915                   /usr/local/lib/python3.5/dist-packages/matplotlib/_path.cpython-35m-x86_64-linux-gnu.so
7fb38c68c000-7fb38c88b000 ---p 0002e000 103:02 1351915                   /usr/local/lib/python3.5/dist-packages/matplotlib/_path.cpython-35m-x86_64-linux-gnu.so
7fb38c88b000-7fb38c88d000 rw-p 0002d000 103:02 1351915                   /usr/local/lib/python3.5/dist-packages/matplotlib/_path.cpython-35m-x86_64-linux-gnu.so
7fb38c88d000-7fb38cb0d000 rw-p 00000000 00:00 0
7fb38cb0d000-7fb38cb12000 r-xp 00000000 103:02 1179411                   /usr/lib/python3.5/lib-dynload/mmap.cpython-35m-x86_64-linux-gnu.so
7fb38cb12000-7fb38cd12000 ---p 00005000 103:02 1179411                   /usr/lib/python3.5/lib-dynload/mmap.cpython-35m-x86_64-linux-gnu.so
7fb38cd12000-7fb38cd13000 r--p 00005000 103:02 1179411                   /usr/lib/python3.5/lib-dynload/mmap.cpython-35m-x86_64-linux-gnu.so
7fb38cd13000-7fb38cd14000 rw-p 00006000 103:02 1179411                   /usr/lib/python3.5/lib-dynload/mmap.cpython-35m-x86_64-linux-gnu.so
7fb38cd14000-7fb38cd1b000 r-xp 00000000 103:02 1179398                   /usr/lib/python3.5/lib-dynload/_csv.cpython-35m-x86_64-linux-gnu.so
7fb38cd1b000-7fb38cf1b000 ---p 00007000 103:02 1179398                   /usr/lib/python3.5/lib-dynload/_csv.cpython-35m-x86_64-linux-gnu.so
7fb38cf1b000-7fb38cf1c000 r--p 00007000 103:02 1179398                   /usr/lib/python3.5/lib-dynload/_csv.cpython-35m-x86_64-linux-gnu.so
7fb38cf1c000-7fb38cf1e000 rw-p 00008000 103:02 1179398                   /usr/lib/python3.5/lib-dynload/_csv.cpython-35m-x86_64-linux-gnu.so
7fb38cf1e000-7fb38cf5e000 rw-p 00000000 00:00 0
7fb38cf5e000-7fb38d042000 r-xp 00000000 103:02 1981464                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/sparse.cpython-35m-x86_64-linux-gnu.so
7fb38d042000-7fb38d242000 ---p 000e4000 103:02 1981464                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/sparse.cpython-35m-x86_64-linux-gnu.so
7fb38d242000-7fb38d247000 rw-p 000e4000 103:02 1981464                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/sparse.cpython-35m-x86_64-linux-gnu.so
7fb38d247000-7fb38d248000 rw-p 00000000 00:00 0
7fb38d248000-7fb38d286000 r-xp 00000000 103:02 1981485                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/internals.cpython-35m-x86_64-linux-gnu.so
7fb38d286000-7fb38d485000 ---p 0003e000 103:02 1981485                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/internals.cpython-35m-x86_64-linux-gnu.so
7fb38d485000-7fb38d48a000 rw-p 0003d000 103:02 1981485                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/internals.cpython-35m-x86_64-linux-gnu.so
7fb38d48a000-7fb38d50b000 rw-p 00000000 00:00 0
7fb38d50b000-7fb38d514000 r-xp 00000000 103:02 1981481                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/indexing.cpython-35m-x86_64-linux-gnu.so
7fb38d514000-7fb38d713000 ---p 00009000 103:02 1981481                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/indexing.cpython-35m-x86_64-linux-gnu.so
7fb38d713000-7fb38d714000 rw-p 00008000 103:02 1981481                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/indexing.cpython-35m-x86_64-linux-gnu.so
7fb38d714000-7fb38d815000 rw-p 00000000 00:00 0
7fb38d815000-7fb38da19000 r-xp 00000000 103:02 1981469                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/interval.cpython-35m-x86_64-linux-gnu.so
7fb38da19000-7fb38dc19000 ---p 00204000 103:02 1981469                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/interval.cpython-35m-x86_64-linux-gnu.so
7fb38dc19000-7fb38dc28000 rw-p 00204000 103:02 1981469                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/interval.cpython-35m-x86_64-linux-gnu.so
7fb38dc28000-7fb38dceb000 rw-p 00000000 00:00 0
7fb38dceb000-7fb38dd02000 r-xp 00000000 103:02 1981480                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/ops.cpython-35m-x86_64-linux-gnu.so
7fb38dd02000-7fb38df01000 ---p 00017000 103:02 1981480                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/ops.cpython-35m-x86_64-linux-gnu.so
7fb38df01000-7fb38df04000 rw-p 00016000 103:02 1981480                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/ops.cpython-35m-x86_64-linux-gnu.so
7fb38df04000-7fb38e110000 r-xp 00000000 103:02 1981479                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/join.cpython-35m-x86_64-linux-gnu.so
7fb38e110000-7fb38e310000 ---p 0020c000 103:02 1981479                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/join.cpython-35m-x86_64-linux-gnu.so
7fb38e310000-7fb38e319000 rw-p 0020c000 103:02 1981479                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/join.cpython-35m-x86_64-linux-gnu.so
7fb38e319000-7fb38e35b000 rw-p 00000000 00:00 0
7fb38e35b000-7fb38e3b2000 r-xp 00000000 103:02 1981497                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/offsets.cpython-35m-x86_64-linux-gnu.so
7fb38e3b2000-7fb38e5b2000 ---p 00057000 103:02 1981497                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/offsets.cpython-35m-x86_64-linux-gnu.so
7fb38e5b2000-7fb38e5ba000 rw-p 00057000 103:02 1981497                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/offsets.cpython-35m-x86_64-linux-gnu.so
7fb38e5ba000-7fb38e5fb000 rw-p 00000000 00:00 0
7fb38e5fb000-7fb38e63d000 r-xp 00000000 103:02 1981499                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/resolution.cpython-35m-x86_64-linux-gnu.so
7fb38e63d000-7fb38e83d000 ---p 00042000 103:02 1981499                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/resolution.cpython-35m-x86_64-linux-gnu.so
7fb38e83d000-7fb38e842000 rw-p 00042000 103:02 1981499                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/resolution.cpython-35m-x86_64-linux-gnu.so
7fb38e842000-7fb38e843000 rw-p 00000000 00:00 0
7fb38e843000-7fb38e861000 r-xp 00000000 103:02 1981489                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/frequencies.cpython-35m-x86_64-linux-gnu.so
7fb38e861000-7fb38ea61000 ---p 0001e000 103:02 1981489                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/frequencies.cpython-35m-x86_64-linux-gnu.so
7fb38ea61000-7fb38ea64000 rw-p 0001e000 103:02 1981489                   /usr/local/lib/python3.5/dist-packages/pandas/_libs/tslibs/frequencies.cpython-35m-x86_64-linux-gnu.soAborted (core dumped)

```